src/js/App/Header/AppFilter.js

```diff
@@ -96,33 +98,42 @@ const AppFilter = () => {
         ouiaId="App Filter"
       >
         <div className="content">
-          <Flex className="search">
-            <SearchInput
-              placeholder="Find application or service"
-              value={filterValue}
-              onChange={(val) => setFilterValue(val)}
-              onClear={() => setFilterValue('')}
-            />
-          </Flex>
-          {filteredApps?.length > 0 ? (
-            <div className="gallery">
-              {filteredApps.map((app) => (
-                <App key={app.id} {...app} />
-              ))}
-            </div>
+          {isLoaded ? (
+            <React.Fragment>
+              <Flex className="search">
+                <SearchInput
+                  placeholder="Find application or service"
+                  value={filterValue}
+                  onChange={(val) => setFilterValue(val)}
+                  onClear={() => setFilterValue('')}
+                />
+              </Flex>
+              {filteredApps?.length > 0 ? (
+                <div className="gallery">
+                  {filteredApps.map((app) => (
+                    <App key={app.id} {...app} />
+                  ))}
+                </div>
+              ) : (
+                <EmptyState className="pf-u-mt-xl" variant={EmptyStateVariant.full}>
+                  <EmptyStateIcon className="pf-u-mb-xl" icon={FilterIcon} />
+                  <Title headingLevel="h4">No matching applications or services found.</Title>
+                  <EmptyStateBody className="pf-u-mb-xl">
+                    This filter criteria matches no applications or services. Try changing your input filter.
+                  </EmptyStateBody>
+                  <Button className="pf-u-mt-lg" variant="link" onClick={() => setFilterValue('')}>
+                    Clear all filters
+                  </Button>
+                </EmptyState>
+              )}
+            </React.Fragment>
           ) : (
-            <EmptyState className="pf-u-mt-xl" variant={EmptyStateVariant.full}>
-              <EmptyStateIcon className="pf-u-mb-xl" icon={FilterIcon} />
-              <Title headingLevel="h4">No matching applications or services found.</Title>
-              <EmptyStateBody className="pf-u-mb-xl">
-                This filter criteria matches no applications or services. Try changing your input filter.
-              </EmptyStateBody>
-              <Button className="pf-u-mt-lg" variant="link" onClick={() => setFilterValue('')}>
-                Clear all filters
-              </Button>
-            </EmptyState>
+            <Bullseye>
```

Comment:
Can you add some padding modifiers? The spinners are a bit wonky and can add scrollbars all over the parent container. We usually use `pf-u-p-lg`. You can check or use the LoadingComponent node.

Reply:
Yep, `pf-u-p-lg` wasn't enough so I used `pf-u-p-xl`